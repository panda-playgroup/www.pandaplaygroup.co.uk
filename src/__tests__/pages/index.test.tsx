import { render } from "@testing-library/react";
import IndexPage, { Head } from "../../pages";

describe("index", () => {
    it("renders", () => {
        const { asFragment } = render(<IndexPage />);

        expect(asFragment()).toMatchSnapshot();
    });
});

describe("head", () => {
    it("renders", () => {
        const { asFragment } = render(<Head />);

        expect(asFragment()).toMatchSnapshot();
    });
});
