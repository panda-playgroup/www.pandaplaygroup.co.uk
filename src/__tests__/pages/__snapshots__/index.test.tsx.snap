// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index renders 1`] = `
<DocumentFragment>
  <main
    style="color: rgb(35, 33, 41); padding: 96px; font-family: -apple-system, Roboto, sans-serif, serif;"
  >
    <h1
      class="text-blue-600"
      style="margin-top: 0px; margin-bottom: 64px; max-width: 320px;"
    >
      Congratulations
      <br />
      <span
        style="color: rgb(102, 51, 153);"
      >
        â€” you just made a Gatsby site! ðŸŽ‰ðŸŽ‰ðŸŽ‰
      </span>
    </h1>
    <p
      style="margin-bottom: 48px;"
    >
      Edit 
      <code
        style="color: rgb(138, 101, 52); padding: 4px; background-color: rgb(255, 244, 219); font-size: 1.25rem; border-radius: 4px;"
      >
        src/pages/index.tsx
      </code>
       to see this page update in real-time. ðŸ˜Ž
    </p>
    <p
      style="margin-bottom: 48px;"
    />
    <div
      class="flex flex-col gap-2 p-4 text-sm text-cyan-700 bg-cyan-100 border-cyan-500 dark:bg-cyan-200 dark:text-cyan-800 rounded-lg"
      role="alert"
    >
      <div
        class="flex items-center"
        data-testid="flowbite-alert-wrapper"
      >
        <div>
          This is a Flowbite alert.
        </div>
      </div>
    </div>
    <p />
    <p
      style="margin-bottom: 48px;"
    >
      <button
        aria-expanded="false"
        aria-haspopup="menu"
        class="group flex h-min items-center justify-center p-0.5 text-center font-medium focus:z-10 focus:outline-none text-white bg-cyan-700 border border-transparent enabled:hover:bg-cyan-800 focus:ring-cyan-300 dark:bg-cyan-600 dark:enabled:hover:bg-cyan-700 dark:focus:ring-cyan-800 rounded-lg focus:ring-2 w-fit"
        data-testid="flowbite-dropdown-target"
        id=":r1:"
        type="button"
      >
        <span
          class="flex items-center rounded-md text-sm px-4 py-2"
        >
          Dropdown button
          <svg
            aria-hidden="true"
            class="ml-2 h-4 w-4"
            fill="none"
            height="1em"
            stroke="currentColor"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 9l-7 7-7-7"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </span>
      </button>
    </p>
    <div
      style="height: 250px; width: 550px; margin-bottom: 48px;"
    >
      <div
        class="relative h-full w-full"
        data-testid="carousel"
      >
        <div
          class="flex h-full snap-mandatory overflow-y-hidden overflow-x-scroll scroll-smooth rounded-lg snap-x indiana-scroll-container indiana-scroll-container--hide-scrollbars"
        >
          <div
            class="w-full flex-shrink-0 transform cursor-grab snap-center"
            data-active="true"
            data-testid="carousel-item"
          >
            <img
              alt="..."
              class="absolute top-1/2 left-1/2 block w-full -translate-x-1/2 -translate-y-1/2"
              src="https://flowbite.com/docs/images/carousel/carousel-1.svg"
            />
          </div>
          <div
            class="w-full flex-shrink-0 transform cursor-grab snap-center"
            data-active="false"
            data-testid="carousel-item"
          >
            <img
              alt="..."
              class="absolute top-1/2 left-1/2 block w-full -translate-x-1/2 -translate-y-1/2"
              src="https://flowbite.com/docs/images/carousel/carousel-2.svg"
            />
          </div>
          <div
            class="w-full flex-shrink-0 transform cursor-grab snap-center"
            data-active="false"
            data-testid="carousel-item"
          >
            <img
              alt="..."
              class="absolute top-1/2 left-1/2 block w-full -translate-x-1/2 -translate-y-1/2"
              src="https://flowbite.com/docs/images/carousel/carousel-3.svg"
            />
          </div>
          <div
            class="w-full flex-shrink-0 transform cursor-grab snap-center"
            data-active="false"
            data-testid="carousel-item"
          >
            <img
              alt="..."
              class="absolute top-1/2 left-1/2 block w-full -translate-x-1/2 -translate-y-1/2"
              src="https://flowbite.com/docs/images/carousel/carousel-4.svg"
            />
          </div>
          <div
            class="w-full flex-shrink-0 transform cursor-grab snap-center"
            data-active="false"
            data-testid="carousel-item"
          >
            <img
              alt="..."
              class="absolute top-1/2 left-1/2 block w-full -translate-x-1/2 -translate-y-1/2"
              src="https://flowbite.com/docs/images/carousel/carousel-5.svg"
            />
          </div>
        </div>
        <div
          class="absolute bottom-5 left-1/2 flex -translate-x-1/2 space-x-3"
        >
          <button
            aria-label="Slide 1"
            class="h-3 w-3 rounded-full bg-white dark:bg-gray-800"
            data-testid="carousel-indicator"
          />
          <button
            aria-label="Slide 2"
            class="h-3 w-3 rounded-full bg-white/50 hover:bg-white dark:bg-gray-800/50 dark:hover:bg-gray-800"
            data-testid="carousel-indicator"
          />
          <button
            aria-label="Slide 3"
            class="h-3 w-3 rounded-full bg-white/50 hover:bg-white dark:bg-gray-800/50 dark:hover:bg-gray-800"
            data-testid="carousel-indicator"
          />
          <button
            aria-label="Slide 4"
            class="h-3 w-3 rounded-full bg-white/50 hover:bg-white dark:bg-gray-800/50 dark:hover:bg-gray-800"
            data-testid="carousel-indicator"
          />
          <button
            aria-label="Slide 5"
            class="h-3 w-3 rounded-full bg-white/50 hover:bg-white dark:bg-gray-800/50 dark:hover:bg-gray-800"
            data-testid="carousel-indicator"
          />
        </div>
        <div
          class="absolute top-0 left-0 flex h-full items-center justify-center px-4 focus:outline-none"
        >
          <button
            aria-label="Previous slide"
            class="group"
            data-testid="carousel-left-control"
            type="button"
          >
            <span
              class="inline-flex h-8 w-8 items-center justify-center rounded-full bg-white/30 group-hover:bg-white/50 group-focus:outline-none group-focus:ring-4 group-focus:ring-white dark:bg-gray-800/30 dark:group-hover:bg-gray-800/60 dark:group-focus:ring-gray-800/70 sm:h-10 sm:w-10"
            >
              <svg
                aria-hidden="true"
                class="h-5 w-5 text-white dark:text-gray-800 sm:h-6 sm:w-6"
                fill="none"
                height="1em"
                stroke="currentColor"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M15 19l-7-7 7-7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
            </span>
          </button>
        </div>
        <div
          class="absolute top-0 right-0 flex h-full items-center justify-center px-4 focus:outline-none"
        >
          <button
            aria-label="Next slide"
            class="group"
            data-testid="carousel-right-control"
            type="button"
          >
            <span
              class="inline-flex h-8 w-8 items-center justify-center rounded-full bg-white/30 group-hover:bg-white/50 group-focus:outline-none group-focus:ring-4 group-focus:ring-white dark:bg-gray-800/30 dark:group-hover:bg-gray-800/60 dark:group-focus:ring-gray-800/70 sm:h-10 sm:w-10"
            >
              <svg
                aria-hidden="true"
                class="h-5 w-5 text-white dark:text-gray-800 sm:h-6 sm:w-6"
                fill="none"
                height="1em"
                stroke="currentColor"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 5l7 7-7 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
            </span>
          </button>
        </div>
      </div>
    </div>
    <ul
      style="padding-left: 0px;"
    >
      <li
        style="color: rgb(137, 84, 168); font-weight: bold; font-size: 16px; vertical-align: 5%; list-style-type: none; display: inline-block; margin-bottom: 24px; margin-right: 12px;"
      >
        <a
          href="https://www.gatsbyjs.com/docs/how-to/custom-configuration/typescript/?utm_source=starter&utm_medium=ts-docs&utm_campaign=minimal-starter-ts"
          style="color: rgb(137, 84, 168); font-weight: bold; font-size: 16px; vertical-align: 5%;"
        >
          TypeScript Documentation
        </a>
      </li>
      <li
        style="color: rgb(137, 84, 168); font-weight: bold; font-size: 16px; vertical-align: 5%; list-style-type: none; display: inline-block; margin-bottom: 24px; margin-right: 12px;"
      >
        <a
          href="https://www.gatsbyjs.com/docs/how-to/local-development/graphql-typegen/?utm_source=starter&utm_medium=ts-docs&utm_campaign=minimal-starter-ts"
          style="color: rgb(137, 84, 168); font-weight: bold; font-size: 16px; vertical-align: 5%;"
        >
          GraphQL Typegen Documentation
        </a>
      </li>
    </ul>
    <ul
      style="margin-bottom: 96px; padding-left: 0px;"
    >
      <li
        style="font-weight: 300; font-size: 24px; max-width: 560px; margin-bottom: 30px; color: rgb(233, 88, 0);"
      >
        <span>
          <a
            href="https://www.gatsbyjs.com/docs/tutorial/getting-started/?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter-ts"
            style="color: rgb(137, 84, 168); font-weight: bold; font-size: 16px; vertical-align: 5%;"
          >
            Tutorial
          </a>
          <p
            style="color: rgb(35, 33, 41); font-size: 14px; margin-top: 10px; margin-bottom: 0px; line-height: 1.25;"
          >
            A great place to get started if you're new to web development. Designed to guide you through setting up your first Gatsby site.
          </p>
        </span>
      </li>
      <li
        style="font-weight: 300; font-size: 24px; max-width: 560px; margin-bottom: 30px; color: rgb(16, 153, 168);"
      >
        <span>
          <a
            href="https://www.gatsbyjs.com/docs/how-to/?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter-ts"
            style="color: rgb(137, 84, 168); font-weight: bold; font-size: 16px; vertical-align: 5%;"
          >
            How to Guides
          </a>
          <p
            style="color: rgb(35, 33, 41); font-size: 14px; margin-top: 10px; margin-bottom: 0px; line-height: 1.25;"
          >
            Practical step-by-step guides to help you achieve a specific goal. Most useful when you're trying to get something done.
          </p>
        </span>
      </li>
      <li
        style="font-weight: 300; font-size: 24px; max-width: 560px; margin-bottom: 30px; color: rgb(188, 2, 127);"
      >
        <span>
          <a
            href="https://www.gatsbyjs.com/docs/reference/?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter-ts"
            style="color: rgb(137, 84, 168); font-weight: bold; font-size: 16px; vertical-align: 5%;"
          >
            Reference Guides
          </a>
          <p
            style="color: rgb(35, 33, 41); font-size: 14px; margin-top: 10px; margin-bottom: 0px; line-height: 1.25;"
          >
            Nitty-gritty technical descriptions of how Gatsby works. Most useful when you need detailed information about Gatsby's APIs.
          </p>
        </span>
      </li>
      <li
        style="font-weight: 300; font-size: 24px; max-width: 560px; margin-bottom: 30px; color: rgb(13, 150, 242);"
      >
        <span>
          <a
            href="https://www.gatsbyjs.com/docs/conceptual/?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter-ts"
            style="color: rgb(137, 84, 168); font-weight: bold; font-size: 16px; vertical-align: 5%;"
          >
            Conceptual Guides
          </a>
          <p
            style="color: rgb(35, 33, 41); font-size: 14px; margin-top: 10px; margin-bottom: 0px; line-height: 1.25;"
          >
            Big-picture explanations of higher-level Gatsby concepts. Most useful for building understanding of a particular topic.
          </p>
        </span>
      </li>
      <li
        style="font-weight: 300; font-size: 24px; max-width: 560px; margin-bottom: 30px; color: rgb(142, 184, 20);"
      >
        <span>
          <a
            href="https://www.gatsbyjs.com/plugins?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter-ts"
            style="color: rgb(137, 84, 168); font-weight: bold; font-size: 16px; vertical-align: 5%;"
          >
            Plugin Library
          </a>
          <p
            style="color: rgb(35, 33, 41); font-size: 14px; margin-top: 10px; margin-bottom: 0px; line-height: 1.25;"
          >
            Add functionality and customize your Gatsby site or app with thousands of plugins built by our amazing developer community.
          </p>
        </span>
      </li>
      <li
        style="font-weight: 300; font-size: 24px; max-width: 560px; margin-bottom: 30px; color: rgb(102, 51, 153);"
      >
        <span>
          <a
            href="https://www.gatsbyjs.com/cloud?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter-ts"
            style="color: rgb(137, 84, 168); font-weight: bold; font-size: 16px; vertical-align: 5%;"
          >
            Build and Host
          </a>
          <span
            aria-label="New Badge"
            style="color: rgb(255, 255, 255); background-color: rgb(8, 132, 19); border: 1px solid #088413; font-size: 11px; font-weight: bold; letter-spacing: 1px; border-radius: 4px; padding: 4px 6px; display: inline-block; position: relative; top: -2px; margin-left: 10px; line-height: 1;"
          >
            NEW!
          </span>
          <p
            style="color: rgb(35, 33, 41); font-size: 14px; margin-top: 10px; margin-bottom: 0px; line-height: 1.25;"
          >
            Now youâ€™re ready to show the world! Give your Gatsby site superpowers: Build and host on Gatsby Cloud. Get started for free!
          </p>
        </span>
      </li>
    </ul>
  </main>
</DocumentFragment>
`;
